tests/nested/manual/core20-early-config/task.yaml

```diff
@@ -64,6 +68,9 @@ execute: |
         tests.nested exec "cat /var/snap/pc/common/debug.txt" | MATCH "hostname: foo"
         tests.nested exec "cat /etc/hostname" | MATCH "foo"
         tests.nested exec "hostname" | MATCH "foo"
+
+        # netplan config
+        tests.nested exec "cat /etc/netplan/0-snapd-defaults.yaml" | MATCH br54
```

Comment:
can we somehow check that setting dhcp4 to false or similar works as expected at this point?

Reply:
I added two more tests for this, one that uses a static IP during boot and one that checks that incremental network settings work.

Reply:
the extra testing is good, but is different from what I was thinking. My question was to test that setting later on top of the defaults works

Reply:
@pedronis oh, sorry again that I misunderstood - I added extra checks for this now as well. Good catch!

Reply:
mvo's test cases look good to me. I can confirm that an interface definition in `0-snapd-defaults.yaml` of the same name as an interface definition in `90-snapd-config.yaml` would be overridden (the `90-snapd-config.yaml` settings be in place).
And that is being tested in the `# and updating netplan works` section.

@pedronis Is there anything more specific that you wanted me to have a look at?